ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -18,6 +19,87 @@ import {
   resolvePendingApproval,
 } from '../../../../../store/actions';
 import { confirmSelector } from '../../../selectors';
+import useAlerts from '../../../../../hooks/useAlerts';
+import { MultipleAlertModal } from '../../../../../components/app/confirmations/alerts/multiple-alert-modal';
+import {
+  BackgroundColor,
+  Severity,
+} from '../../../../../helpers/constants/design-system';
+import { getSeverityBackground } from '../../../../../components/app/confirmations/alerts/alert-utils';
+import { Alert } from '../../../../../ducks/confirm-alerts/confirm-alerts';
+
+function ReviewAlertButton({
+  backgroundColor,
+  setAlertModalVisible,
+}: {
+  backgroundColor: BackgroundColor;
+  setAlertModalVisible: (visible: boolean) => void;
+}) {
+  const t = useI18nContext();
+  return (
+    <Button
+      block
+      backgroundColor={backgroundColor}
+      onClick={() => setAlertModalVisible(true)}
+      startIconName={IconName.SecuritySearch}
+      size={ButtonSize.Lg}
+      data-testid="review-alert-button"
+    >
+      {t('reviewAlerts')}
+    </Button>
+  );
+}
+
+function ConfirmFooterButton({
```

Comment:
Since we're already in the footer file, should we just call this `ConfirmButton`?